src/components/Gitter/Gitter.jsx

```diff
@@ -14,7 +14,9 @@ export default class Gitter extends React.Component {
   render() {
     return (
       <div className="gitter">
-        <div className="gitter__button js-gitter-toggle-chat-button">
+        <div
+          className="gitter__button"
+          onClick={() => window.gitterSidecar && window.gitterSidecar.toggleChat(true)}>
```

Comment:
You think the imperative call is more intuitive than the class binding?

Reply:
I think I'm ok with this change but the module-level variable makes more sense to me and would shorten this call:

``` js
onClick={() => sidecar && sidecar.toggleChat(true))
```

Reply:
Yeah i think we can give it a `let` in the top of `Gitter.jsx`